dist/demos/components/modal.html

```diff
@@ -103,11 +103,13 @@ <h1>A11y Dialog</h1>
 
 </div>
 
-<div class="dg_modal hidden" id="my-accessible-dialog" role="dialog" aria-labelledby="my-accessible-dialog_label" aria-modal="true">
+<div class="dg_modal hidden" data-dismissible id="my-accessible-dialog" role="dialog" aria-labelledby="my-accessible-dialog_label" aria-modal="true">
```

Comment:
Question regarding `data-dismissible`. Should this be set equal to a boolean? I think in this case it would be `data-dismissible="true"` or am I not interpreting the functionality correctly?

Reply:
I did not see data-dismissible inside your div in your previous PR but it was working on clickevent.  I know by default it sets to true but I see no boolean value chaning when click event is trigger. How does it work.

Reply:
if you take a look at these two lines https://github.com/baltimorecounty/dotgov-javascript/blob/45e12e262265a8e6b18a45fc924a3ea6846831cf/src/components/modal.js#L15 and https://github.com/baltimorecounty/dotgov-javascript/blob/45e12e262265a8e6b18a45fc924a3ea6846831cf/src/components/modal.js#L31

You will see that we are just checking the existence of this property. We could do data-dismissible="true" and data-dismissible="false" but we wouldn't really need that.

Although you bring up a good opint and maybe this code should be a little more flexible to handle situation.

Reply:
@sgrg1 you convinced me to simplify the usage and the code. data-dismissible will not require true or false.

See the updated docs [here](https://github.com/baltimorecounty/dotgov-javascript/pull/13/files#diff-81c644c2f6f1e88563e968e9a2d17b72R17).

Does that make sense?